pages/index.js

```diff
@@ -1,41 +1,19 @@
-import { StyleSheet, Text, View } from 'react-native'
+import { StyleSheet, View } from "react-native";
+import React from "react";
+import Game from "./tipsy/game.jsx";
 
-export default function App(props) {
-  return (
-    <View style={styles.container}>
-      <Text accessibilityRole="header" style={styles.text}>
-        React Native for Web & Next.js
-      </Text>
-
-      <Text style={styles.link} accessibilityRole="link" href={`/alternate`}>
-        A universal link
-      </Text>
-
-      <View style={styles.textContainer}>
-        <Text accessibilityRole="header" aria-level="2" style={styles.text}>
-          Subheader
-        </Text>
-      </View>
-    </View>
-  )
+export default function App() {
+    return (
+        <View style={styles.container}>
+            <Game></Game>
```

Comment:
```suggestion
            <Game />
```
Unless you plan to have children later?